app/views/admin/people/new.html.slim

```diff
@@ -0,0 +1,6 @@
+- content_for :title
+= t("people.shared.new")
```

Comment:
This can/should be one line I believe. See other examples in app. (I know it's not consistent but we should move toward one line (@emitche please note)